spec/cypress/integration/admin/DeliveryPartnerManagement.feature

```diff
@@ -55,35 +55,37 @@ Feature: Admin user modifying delivery partners
 
   Scenario: It should allow changing name choice during creation
     When I click on "create delivery partner button"
-    And I type "wrong name" into "delivery partner name" field
+    And I type "wrong name" into "delivery partner name input"
     And I click the submit button
     And I click on "Lead Provider 1" label
     And I click the submit button
     And I click on "2021" label
     And I click the submit button
     And I click on "change name link"
-    And I type "{selectall}New delivery partner" into "delivery partner name" field
+    And I clear "delivery partner name input"
+    And I type "New delivery partner" into "delivery partner name input"
     And I click the submit button
     And I click the submit button
     And I click the submit button
-    Then "main" should contain "New delivery partner"
+    Then "page body" should contain "New delivery partner"
 
     When I click the submit button
     Then I should be on "delivery partner index" page
-    And "main" should contain "New delivery partner"
+    And "page body" should contain "New delivery partner"
 
   Scenario: Admins should be able to edit delivery partners
     When I click on "a" containing "Delivery Partner 1"
     Then I should be on "delivery partner edit" page
     And "name input" should have value "Delivery Partner 1"
     # And the page should be accessible
 
-    When I type "{selectall}New delivery partner" into "name input" field
+    When I clear "name input"
+    And I type "New delivery partner" into "name input"
```

Comment:
Wrong input I think

Reply:
What do you mean?

Reply:
Hmm, I thought it should be `"delivery partner name input"`. But maybe not, because these are different pages...

Reply:
Ah I just simplified https://github.com/DFE-Digital/early-careers-framework/pull/166/files#diff-205f09c150777a69c6aefe2dd47f2206e4ecb46b977e5494bcd55326f348fd32L18

`"delivery partner name input"` would work too, because the selector is literally just `input[type="text"]` 😆